script.js

```diff
@@ -0,0 +1,85 @@
+var userName = prompt("What is your name?");
+var foodGroup = prompt("What do you want today?\n1. Pizza\n2. Pasta\n3. Salad\nPlease write the number of your choice (1, 2, or 3).");
+var foodSubGroup;
+var foodPrice;
+var childFoodPrice;
+
+// Validate foodGroup input
+if (foodGroup === "1" || foodGroup === "2" || foodGroup === "3") {
+  switch (foodGroup) {
+    case "1":
+      foodGroup = "pizza";
+      break;
+    case "2":
+      foodGroup = "pasta";
+      break;
+    case "3":
+      foodGroup = "salad";
+      break;
+  }
+
+  alert(`You have chosen ${foodGroup}. An excellent choice!`);
+
+  // Prompt for food subgroup based on food group
+  if (foodGroup === "pizza") {
+    foodSubGroup = prompt("What kind of pizza would you like?\n1. Capricciosa\n2. Calzone\n3. Quattro Stagioni\nPlease write the number of your choice (1, 2, or 3).");
+    switch (foodSubGroup) {
+      case "1":
+        foodSubGroup = "capricciosa";
+        foodPrice = 100;
+        break;
+      case "2":
+        foodSubGroup = "calzone";
+        foodPrice = 120;
+        break;
+      case "3":
+        foodSubGroup = "quattro stagioni";
+        foodPrice = 150;
+        break;
+    }
```

Comment:
Something to ponder upon: would it be a good idea to have a "`default case`" at the end of your switches?
Here for example, letters or integers>3 as values for `foodSubGroup` do not prompt any error message, and your code runs to the end with weird values (the random ones entered appear in your final message: `You are about to order M.  The cost will be undefined kronor`).